Description: Remove bundler usage from test
 Remove the usage of Bundler from test helper file
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/helper.rb
+++ b/tests/helper.rb
@@ -1,13 +1,5 @@
-begin
-  require "codeclimate-test-reporter"
-  CodeClimate::TestReporter.start
-rescue LoadError => e
-  $stderr.puts "not recording test coverage: #{e.inspect}"
-end
-
 require File.expand_path('../../lib/fog/local', __FILE__)
 
-Bundler.require(:test)
 require 'tmpdir'
 
 Excon.defaults.merge!(:debug_request => true, :debug_response => true)
